ramenpos/RamenPos/View Controllers/Main/MainViewController+PrintStatusActions.swift

```diff
@@ -237,10 +237,10 @@ extension MainViewController {
         if (txState.response != nil) {
             let gltResponse = SPIGetLastTransactionResponse(message: txState.response)
             
-            if (_lastCmd.count > 1) {
+            if (txtReferenceId.text != "") {
```

Comment:
isEmpty can be helpful here just FYI (no need to change, no biggie):
```suggestion
            if (!txtReferenceId.text.isEmpty) {
```

Reply:
Thanks 👍 :)